CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 * [#7977](https://github.com/rubocop/rubocop/issues/7977): Add `Layout/RedundantLineBreak` cop. ([@jonas054][])
 * [#9691](https://github.com/rubocop/rubocop/issues/9691): Add configuration parameter `InspectBlocks` to `Layout/RedundantLineBreak`. ([@jonas054][])
 
+### Bug fixes
```

Comment:
Seems you really don't like the new way to add changelog entries. 😆

Reply:
On the contrary. I like it a lot. I just forgot that it existed. :slightly_smiling_face: